docs/Erlang Distribution.md

```diff
@@ -0,0 +1,92 @@
+# Erlang Distribution
+
+## Static Configuration
+
+The simplest way to configure Erlang distribution is to statically (i.e., at
+compile time) set the node name and cookie. This is done by editing the
+`rel/vm.args` file, which contains the command-line arguments used to launch
+the Erlang/OTP virtual machine.
+
+You would simply add `-name` and `-setcookie` parameters, as follows:
```

Comment:
Out of curiousity, how does that work, then? If I have a Nerves node and give it the name `nerves.local`, then how does my laptop do the DNS resolution when I refer to `nerves.local` from it? I thought that the only two options were DNS and mDNS.

`-sname` can be used between hosts. I actually use it more frequently since I don't want to deal with the fully qualified names. The hostname part of the node name has always felt like a dynamic so I never wanted to lock it down to a fully qualified name. It occurs to me that your PR to change the node name at runtime might just be another way of addressing the same issue.